### ISSUE TITLE
Darkmode

### ISSUE THREAD

**Issue by JNSMDT** [angertitan/cryptosubtle-pw-generator#22]
With the new Designsystem (#21) the first new addition should be a real dark mode since many extension just turn the password field to a dark color.
